packages/engine-ui/src/components/CreateBooking.js

```diff
@@ -38,18 +38,35 @@ const CreateBooking = ({ onSubmit }) => {
   const history = useHistory()
   const [isFinished, setIsFinished] = React.useState(false)
   const [formState, setState] = React.useState(initialState)
+  const [showErrorMessage, setErrorState] = React.useState({
+    pickup: false,
+    delivery: false,
+  })
   const setUIState = stores.ui((state) => state.dispatch)
 
   hooks.useFormStateWithMapClickControl('pickup', 'delivery', setState)
 
+  const validateAddresses = () => {
+    const pickup = !formState.pickup.lat || !formState.pickup.lon
+    const delivery = !formState.delivery.lat || !formState.delivery.lon
```

Comment:
I also suggest a naming change here. 
```suggestion
    const isValidPickup = formState.pickup.lat && formState.pickup.lon
    const isValidDelivery = formState.delivery.lat && formState.delivery.lon
```